data-pipeline/csv-validation/petl-validation.py

```diff
@@ -0,0 +1,51 @@
+
+# Exception Classes
+class InvalidStateException(Exception):
```

Comment:
These exceptions are _fine_, but for more custom errors than `InvalidStateException` or `InvalidJurisdictionException` check: https://stackoverflow.com/a/26938914/6482196